src/articles/_posts/2009-03-06-javascript-best-practices.md

```diff
@@ -546,26 +547,27 @@ As I am using the generic — really throw-away — variable names `ul` and `li`
 
 	function renderProfiles(o) {
 		var out = document.getElementById('profiles');
+		var ul = document.createElement('ul');
 		for(var i = 0; i < o.members.length; i++) {
-			var ul = document.createElement('ul');
 			var li = document.createElement('li');
-			li.appendChild(document.createTextNode(data.members[i].name));
-			li.appendChild(addMemberData(o.members[i]));
+			li.appendChild(document.createTextNode(o.members[i].name));
+			li.appendChild(addMemberData(o.members[i].data));
+			ul.appendChild(li);
 		}
 		out.appendChild(ul);
 	}
-	function addMemberData(member) {
+	function addMemberData(data) {
```

Comment:
Changes to reflect that only the `data` member is being passed now.
All uses of `member.data` changed to just `data`.